{
    "cveId": "CVE-2023-42962",
    "version": "1.0.0",
    "timestamp": "2024-12-06T11:32:07.789868+00:00",
    "description": "This issue was addressed with improved checks This issue is fixed in iOS 17.2 and iPadOS 17.2, iOS 16.7.3 and iPadOS 16.7.3. A remote attacker may be able to cause a denial-of-service.",
    "keyphrases": {
        "rootcause": "",
        "weakness": "",
        "impact": "denial-of-service",
        "vector": "",
        "attacker": "remote attacker",
        "product": [
            "iOS",
            "iPadOS"
        ],
        "version": [
            "17.2",
            "17.2",
            "16.7.3",
            "16.7.3"
        ],
        "component": ""
    }
}
